src/demo.js

```diff
@@ -2,11 +2,12 @@
 
 import React, {Component} from 'react';
 import moment from 'moment';
-
+import _ from 'lodash';
 import Timeline from './timeline';
 
 import {Layout, Form, InputNumber, Button, DatePicker} from 'antd';
 import 'antd/dist/antd.css';
+import TimelineItem from 'antd/lib/timeline/TimelineItem';
```

Comment:
Do we need this?

Reply:
I didn't add that... i swear. But git says otherwise :(